namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-staging/resources/main.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "${var.aws_region}"
+}
```

Comment:
You need the `aws.london` provider here, for the latest version of the RDS module;

```
provider "aws" {
  alias  = "london"
  region = "eu-west-2"
}
```

See https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/master/example/rds.tf